core/metacling/src/TClingClassInfo.cxx

```diff
@@ -162,6 +162,8 @@ long TClingClassInfo::ClassProperty() const
    // We now have a class or a struct.
    const CXXRecordDecl *CRD =
       llvm::dyn_cast<CXXRecordDecl>(GetDecl());
+   if (!CRD)
```

Comment:
This is likely redundant with the construction of fDecl .... humm ... it is plausible that in some future their might be a new case introduced by a future C++ standard ... so maybe its fine.